axolotl-web/src/pages/MessageList.vue

```diff
@@ -315,6 +359,81 @@ export default {
     scrollDown() {
       document.getElementById("chat-bottom").scrollIntoView();
     },
+    recordAudio(){
```

Comment:
It would be nice to put all this logic in its own file.
MessageList.vue and Message.vue are bloated by functionallity.

Reply:
Next time i will do this